capsules/src/tmp006.rs

```diff
@@ -144,7 +144,8 @@ impl<'a> TMP006<'a> {
 
     fn enable_interrupts(&self) {
         // setup interrupts from the sensor
-        self.interrupt_pin.enable_input(InputMode::PullUp);
+        // TODO(alevy): do we need to make sure it's a pull up?
```

Comment:
Yes, this line does need to be pulled up. The interrupt line form the TMP006 is open-drain and requires a pull-up.